sdk/Lusid.Sdk.Tests/Utilities/DemoInstrumentBase.cs

```diff
@@ -89,7 +89,7 @@ internal void CallLusidGetValuationEndpoint(LusidInstrument instrument, ModelSel
                 // SimpleStatic pricing is lookup pricing. As such, we upsert a quote.
                 // Note that inside CreatePortfolioAndInstrument, the method TestDataUtilities.BuildInstrumentUpsertRequest books the instrument using "ClientInternal".
                 // Hence upsert a quote using ClientInternal as the instrumentIdType.
-                var quoteRequest = TestDataUtilities.BuildQuoteRequest(scope, instrumentID, QuoteSeriesId.InstrumentIdTypeEnum.ClientInternal, 100m, "USD", TestDataUtilities.EffectiveAt);
```

Comment:
Consider wrapping these long lines like this:

var quoteRequest = TestDataUtilities.BuildQuoteRequest(
  scope,
  instrumentID,
  QuoteSeriesId.InstrumentIdTypeEnum.ClientInternal,
  100m,
  "USD",
  TestDataUtilities.EffectiveAt);

Not sure if we have an agreed upon convention, but I typically try to keep lines <120 chars